stdlib/3/tkinter/simpledialog.pyi

```diff
@@ -0,0 +1,27 @@
+from tkinter import Event, Misc, Toplevel
+from typing import Any, List, Optional
+
+class Dialog(Toplevel):
+    def __init__(self, parent: Optional[Misc], title: Optional[str] = ...) -> None: ...
+    def body(self, master) -> None: ...
+    def buttonbox(self): ...
+
+class SimpleDialog:
+    def __init__(
+        self,
+        master: Optional[Misc],
+        text: str = ...,
+        buttons: List[str] = ...,
```

Comment:
I think this should be `typing.Sequence` instead of `List`, or maybe one of the `SupportsFoo` classes in `_typeshed`. It needs to have `__getitem__` and `__len__`, so it could also be e.g. a tuple or a `collections.deque`. (Note that `tkinter._TkinterSequence` is the right type for many things in tkinter, but not for this one.)